packages/big-design/src/components/Search/Search.tsx

```diff
@@ -22,6 +23,7 @@ export const Search: React.FC<TableSearch> = ({ value, onChange, onSubmit }) =>
       <Flex alignItems="center" backgroundColor="white" flexDirection="row" paddingBottom="xxSmall">
         <FlexItem flexGrow={1} marginRight="small">
           <Input
+            aria-label="Search"
```

Comment:
Added the `aria-label` for accessibility. `placeholder` attributes will not be read by screen readers.